mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -42,26 +42,30 @@ def ocn_modelvsobs(config, field, streamMap=None, variableMap=None):
     to their mpas_analysis counterparts.
 
     Authors: Luke Van Roekel, Milena Veneziani, Xylar Asay-Davis
-    Modified: 12/08/2016
+    Modified: 01/29/2017
     """
 
     # read parameters from config file
-    indir = config.get('paths', 'archive_dir_ocn')
+    indir = config.get('input', 'basedir')
 
     streams_filename = config.get('input', 'ocean_streams_filename')
     streams = StreamsFile(streams_filename, streamsdir=indir)
 
     # get a list of timeSeriesStats output files from the streams file,
     # reading only those that are between the start and end dates
-    startDate = config.get('time', 'climo_start_date')
-    endDate = config.get('time', 'climo_end_date')
+    startDate = config.get('climatology', 'start_date')
+    endDate = config.get('climatology', 'end_date')
     streamName = streams.find_stream(streamMap['timeSeriesStats'])
     infiles = streams.readpath(streamName, startDate=startDate,
                                endDate=endDate)
     print 'Reading files {} through {}'.format(infiles[0], infiles[-1])
 
-    plots_dir = config.get('paths', 'plots_dir')
-    obsdir = config.get('paths', 'obs_' + field + 'dir')
+    output_basedir = config.get('output', 'basedir')
+    plots_dir = '{}/{}'.format(output_basedir,
```

Comment:
I think that's a good idea.  Where would you suggest we put this function?  The same question arises  frequently regarding shared functionality that doesn't neatly fit into a category like `io` or `interpolation`.  In this case, though, https://github.com/MPAS-Dev/MPAS-Analysis/blob/develop/mpas_analysis/shared/io/utility.py would make sense, I think.